### ISSUE TITLE
Lazy loading not working

### ISSUE THREAD

**Issue by StasD** [BrainCrumbz/ngtools-webpack-demo#1]
Hello,

So, are you saying that when you run "npm run build-prod" for this project, it actually produces a separate chunk for the PrivateModule module? Because I cloned it to check just that, and this does not work for me...

I was just trying to get lazy loading with AoT work for my project, so far without success (that is, no additional chunks), so I found this your project, but as I said, I'm getting the very same result: no chunks for lazy loaded modules... Very weird...

Using macOS Sierra here with Node v6.10.3. Everything else seems to be coming from "npm install".

**Comment by BrainCrumbz** [comment]
Hi there. Thanks for trying this out.

### tl; dr;

Comment out the part using `ContextReplacementPlugin` in `webpack.config.prod.js`

### Full story

This project is an attempt to hit that target, so having lazy loaded modules working with ngtools/webpack AOT build.

We hit some roadblocks, mainly with ngtools/webpack issues. The last one is tracked by this couple of issues in angular-cli project:

* https://github.com/angular/angular-cli/issues/4431
* https://github.com/angular/angular-cli/issues/6518 (closed for now, but maybe it should be reopened)

The workaround for now, as suggested by angular-cli people, is to comment out that plugin. Actually we're going to update this project with that workaround shortly.

HTH

**Comment by BrainCrumbz** [comment]
This has been fixed (with mentioned workaround) by 68e06e9df30af595a7738a2a99dba8be30119475
